lib/map-store.js

```diff
@@ -96,6 +96,13 @@ MapStore.prototype.transformCoverage = function (coverageMap) {
         return fs.readFileSync(pathutils.asAbsolute(filePath, that.baseDir));
     };
 
+    coverageMap.files().forEach(function (file) {
```

Comment:
I added the test but it actually does not fail... it does not depend on istanbul-lib-instrument. I hope thats fine...